tests/integration/Cargo.toml

```diff
@@ -7,15 +7,21 @@ edition = '2021'
 [dependencies]
 candid = { workspace = true }
 candid_parser = { workspace = true }
+dfx-orbit = { path = "../../tools/dfx-orbit" }
 hex = { workspace = true }
 orbit-essentials = { path = '../../libs/orbit-essentials', version = '0.0.2-alpha.3' }
 ic-cdk = { workspace = true }
 ic-ledger-types = { workspace = true }
 lazy_static = { workspace = true }
 num-bigint = { workspace = true }
 pocket-ic = { workspace = true }
+rand = { workspace = true }
+rand_chacha = { workspace = true }
+reqwest = { version = "0.12", default-features = false }
```

Comment:
nit: can we add this to the root Cargo.toml?

Reply:
Done